x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -4953,129 +4953,6 @@
     "management.settings.offLabel": "Désactivé",
     "management.settings.onLabel": "Activé",
     "management.settings.resetToDefaultLinkText": "Réinitialiser à la valeur par défaut",
-    "monaco.esql.autocomplete.matchingFieldDefinition": "Utiliser pour correspondance avec {matchingField} de la politique",
```

Comment:
Why this PR removes all these translations? 🤔

Reply:
They were part of the feature branch failed checks, so I had to remove them both here and in #171664 .
Now that the autosuggest PR is merged I should be able to merge it and clear this out.

Reply:
Confirm. After latest upstream merge it's gone 🌬️